### ISSUE TITLE
How to unsubscribe from iframeTracker?

### ISSUE THREAD

**Issue by bennyn** [vincepare/iframeTracker-jquery#12]
Hello, given that I have subscribed to some iframes with:

```js
$('iframe.youtube').iframeTracker({
        blurCallback: function(){
            console.info('youtube');
        }
    });
```

What's now the recommended way to unsubscribe from clicks on these iframes?

**Comment by finalclap** [comment]
Hello, there is currently no way to unbind an iframeTracker handler, but I think I will implement it soon since you are not the first user needing this feature.

Basically, to remove the handler, you have two things to do :
* Unbind mouseover & mouseout event handlers bound on the tracked element (`'iframe.youtube'`)
* Remove the handler object from the internal index (`$.iframeTracker.handlersList`)

Otherwise, you can just try something like that (but this is pretty ugly) :

```javascript
$.iframeTracker.disabled = [];
$.iframeTracker.disabled.push('iframe.youtube');

$('iframe.youtube').iframeTracker({
	blurCallback: function(){
		// Check if tracking is enabled
		if ($.iframeTracker.disabled.indexOf('iframe.youtube')) {
			return;
		}
		
		console.info('youtube');
	}
});
```

**Comment by bennyn** [comment]
Hey, thank you for this detailed answer! I will stargaze the project to get notified when it's in. Thank you very much for your great plugin!

**Comment by vincepare** [comment]
The untrack feature has been released (1.0.5) !
Use it like this :
```javascript
$('iframe.youtube').iframeTracker(false);
```

**Comment by bennyn** [comment]
Yaaaaay! I'll update my Bower dependency immediately. Thank you so much! :tada:
